DragaliaAPI.Shared/MasterAsset/MasterAsset.cs

```diff
@@ -265,25 +265,25 @@ public static class MasterAsset
     /// <summary>
     /// Dragon StoryId Arrays indexed by DragonId
     /// </summary>
-    public static MasterAssetData<int, StoryData> DragonStories { get; } =
+    public static readonly MasterAssetData<int, StoryData> DragonStories =
```

Comment:
Could we make it consistent to use get-only public properties instead of fields? I don't think it matters that much since we won't be using custom get set accessors but just better to have public members as properties imo